### PR TITLE
Ft asset sub category #155322115

### PR DESCRIPTION
**Pull request by nelmandela**
### What does this PR do?
  •Creates asset sub category

### Description of work done
  •Add asset sub category model in model.py and tests in test_asset_sub_category_model.py

### How can this be manually tested?
  •python manage.py test

### Relevant pivotal tracker stories
  #155322115

### CODE REVIEW

**Comment by nelmandela** [comment]
Thanks, let me do that

File: api/models.py [hunk 1]
```diff
@@ -21,6 +21,9 @@ class AssetSubCategory(models.Model):
     created_at = models.DateTimeField(auto_now_add=True, editable=False)
     last_modified = models.DateTimeField(auto_now_add=True, editable=False)
 
+    class Meta:
+        verbose_name_plural = 'Asset Categories'
```
Comment (egichuri) [inline api/models.py]: i merged this too soon.. please change this to `Asset Sub Categories`

Comment (nelmandela) [reply api/models.py]: ooops,,,on it

Comment (egichuri) [reply api/models.py]: `Asset Subcategories` ie  😄

Comment (nelmandela) [reply api/models.py]: @egichuri done

Comment (egichuri) [reply api/models.py]: pr?

**Pull request merged**